.github/workflows/pr-title-linter.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint PR Title
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  pr-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: morrisoncole/pr-lint-action@v1.7.0
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          title-regex: "^[a-z]+?\\([a-z0-9-]+?\\): [A-Z]+?-\\d+? \\w.*|^Version Packages|^\\[Snyk\\].*"
+          on-failed-regex-fail-action: true
+          on-failed-regex-create-review: true
+          on-failed-regex-request-changes: true
+          on-failed-regex-comment: "[Lint PR Title] ❌ Failed.\nThe PR title should follow the same convention used for commits, e.g.:\n`type(scope): DSW-123 title` where `DSW-123` is the Jira ticket id or `DSW-000` when one is not available."
+          on-succeeded-regex-dismiss-review-comment: "[Lint PR Title] ✅ All good!"
```

Comment:
Do we really need to add a comment on success? Feels a bit noisy

Reply:
In fact, it only updates the existing one, so no one gets spammed.

Reply:
I appreciate it's only 1 comment but do we really need one if we can already see the check is green?

Reply:
It just feels a little unnecessary and doesn't really add anything useful

Reply:
I think this only shows if it's previously failed - it won't show if it passes first time (looking at the github action).

It looks like if you don't customise this, it'll update it anyway and just display `All good!` so I don't think you can get rid of this behaviour.

Reply:
We have the option of not displaying it at all. If it fails one can check the job output.

I'd say we can turn it off in the near future while we all adapt to the new pattern.

<img width="1045" alt="image" src="https://github.com/justeattakeaway/pie/assets/448801/92f83e59-614b-46a3-9d58-824057de5a6d">